src/fiskaltrust.ifPOS/v1/de/Models/ExportDataResponse.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Serialization;
+using System.Text;
+
+namespace fiskaltrust.ifPOS.v1.de
+{
+    [DataContract]
+    public class ExportDataResponse
+    {
+        [DataMember(Order = 10)]
+        StartExportSessionResponse Session { get; set; }
```

Comment:
IMO we should use the explicit properties instead of this object by sending the token and the serialnumber explicitly

Reply:
changed to 'TokenId' only.